mlir/utils/jenkins/Jenkinsfile

```diff
@@ -29,6 +29,22 @@ void getAndBuildMIOpen(String prefixOpt, String cmakeOpts) {
     buildMIOpen(cmakeOpts)
 }
 
+void buildMIGraphX(String cmakeOpts) {
+    sh '[ ! -d build ] || rm -rf build'
+    cmakeBuild generator: 'Unix Makefiles',\
+        buildDir: 'build',\
+        buildType: 'Release',\
+        installation: 'InSearchPath',\
+        cmakeArgs: ${cmakeOpts}
```

Comment:
You need to quote ${variables} for Jenkins to expand the variable.

Reply:
Oh! fixed now.